apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { Section } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { Divider } from "semantic-ui-react";
+import { AppConstants, history } from "../../core";
+import { getSettingsSectionIcons } from "../../server-configurations";
+import { useTokenReuseConfigData } from "../api";
+
+
+/**
+ * Props for PrivateKey JWT Config Interface settings page.
+ */
+type PrivateKeyJWTConfigPageInterface = IdentifiableComponentInterface;
+
+/**
+ * PrivateKey JWT connector listing page.
+ *
+ * @param props - Props injected to the component.
+ * @returns PrivateKey JWT connector listing page component.
+ */
+export const PrivateKeyJWTConfig: FunctionComponent<PrivateKeyJWTConfigPageInterface> = (
+    props: PrivateKeyJWTConfigPageInterface
+): ReactElement => {
+    const { [ "data-componentid" ]: componentId } = props;
+    const { t } = useTranslation();
+
+    const {
+        data: tokenReuseData,
+        isLoading: isLoading
+    } = useTokenReuseConfigData();
+
+    useEffect(() => {
+        if (isLoading) {
+            return;
+        }
+
+    }, [ isLoading ]);
+
+    /**
+     * Handle connector advance setting selection.
+     */
+    const handleSelection = () => {
+        history.push(AppConstants.getPaths().get("PRIVATE_KEY_JWT_CONFIG_EDIT"));
+    };
+
+    return (
+        <Section
+            data-componentid={ `${componentId}-settings-section` }
+            // eslint-disable-next-line max-len
```

Comment:
Shall we fix the max-length warning

Reply:
fixed with https://github.com/wso2/identity-apps/pull/3786/commits/dd8a9366531a26a1a44411371fe1db17b81925ae